app/src/androidTest/java/com/andryoga/safebox/ui/home/backupAndRestore/BackupAndRestoreWorkersTest.kt

```
@@ -0,0 +1,347 @@
+                bankCardDataRepository,
+                bankAccountDataRepository,
+                secureNoteDataRepository
+            )
+
+            val backupDir = File(context.cacheDir, "backup_tc05")
+            backupDir.deleteRecursively()
+            backupDir.mkdirs()
+
+            // Pre-create 10 backup files (the maximum allowed threshold)
+            for (i in 1..CommonConstants.MAX_BACKUP_FILES) {
+                val formattedIndex = String.format("%02d", i)
```

detekt (1.23.8)

[warning] String.format("%02d", i) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)